types/materializecss__materialize/index.d.ts

```diff
@@ -0,0 +1,40 @@
+// Type definitions for materializecss__materialize 1.1
+// Project: https://materializecss.github.io/materialize/
+// Definitions by:  胡玮文 <https://github.com/huww98>
+//                  Maxim Balaganskiy <https://github.com/MaximBalaganskiy>
+//                  David Moniz <https://github.com/MonizDave>
+//                  Daniel Hoenes <https://github.com/broccoliarchy>
+//                  Rick Lucassen <https://github.com/SnowyLeopard>
+//                  Mauro Mascarenhas <https://github.com/mauromascarenhas/>
```

Comment:
```ts
// Definitions by:  胡玮文 <https://github.com/huww98>
//                  Maxim Balaganskiy <https://github.com/MaximBalaganskiy>
//                  David Moniz <https://github.com/MonizDave>
//                  Daniel Hoenes <https://github.com/broccoliarchy>
//                  Rick Lucassen <https://github.com/SnowyLeopard>
//                  Mauro Mascarenhas <https://github.com/mauromascarenhas/>
```
I don't think this is ok to just copy over peoples handlers here without at least shout out in the PR asking for ACK (or expressing some kind of support in PR comments, etc)

Reply:
Ok, just removed them.